### ISSUE TITLE
Overwriting warnings element?

### ISSUE THREAD

**Issue by rpfisher** [ligovirgo/dqsegdb#43]
```
[rfisher@ldas-pcdev2 ~]$ ligolw_segment_query_dqsegdb -t http://segments.ligo.org --query-segments -a H1:DMT-ANALYSIS_READY:1 -s 1182940864 -e 1182940964
Traceback (most recent call last):
  File "/bin/ligolw_segment_query_dqsegdb", line 750, in <module>
    run_query_segments_dqsegdb(doc, process_id, protocol, server, gps_start_time, gps_end_time, options.include_segments, options.exclude_segments, options.result_name)
  File "/bin/ligolw_segment_query_dqsegdb", line 514, in run_query_segments_dqsegdb
    json_result,queryurl=apicalls.dqsegdbQueryTimesCompatible(protocol,server,ifo,name,version,include_list_string,startTime,endTime)
  File "/usr/lib/python2.7/site-packages/dqsegdb/apicalls.py", line 213, in dqsegdbQueryTimesCompatible
    result=urifunctions.getDataUrllib2(queryurl)
  File "/usr/lib/python2.7/site-packages/dqsegdb/urifunctions.py", line 114, in getDataUrllib2
    warnings.warn("Issue accesing url: %s; Reason: %s" % (url,str(e.reason)))
AttributeError: 'bool' object has no attribute 'warn'
```

**Comment by duncanmmacleod** [comment]
wasn't this fixed by #29?

**Comment by robertbruntz** [comment]
After investigating, this error cannot be reproduced on ldas-pcdev2 on the LHO, LLO, or CIT clusters - all work correctly now - and the issue opener does not remember which system the error was occurring on.  I think there was a time period when the dqsegdb package was not being updated on some systems, so that might have been the cause of the error, but all systems seem to be up-to-date now.  I'm closing this ticket.  If anyone finds a system that still has this bug, please re-open the ticket or start a new one, with details about the system/environment.

**Comment by duncanmmacleod** [comment]
> wasn't this fixed by #29?

Wasn't this fixed by #29 3.5 years ago?

**Comment by robertbruntz** [comment]
Yes - that's the point.  As best I can tell, whichever system Ryan was using in Aug. 2017 wasn't using the newest release (1.5.0, May 2017), so he was getting an error that had already been fixed in the newest release.  Since he doesn't remember which system that was, I went looking for it, and I can't find any systems that aren't running the newest release (1.6.1), so I closed the ticket.
